internal/fs/inode/file.go

```diff
@@ -100,24 +101,25 @@ func NewFileInode(
 	mtimeClock timeutil.Clock) (f *FileInode) {
 	// Set up the basic struct.
 	f = &FileInode{
-		bucket:       bucket,
-		mtimeClock:   mtimeClock,
-		id:           id,
-		name:         name,
-		attrs:        attrs,
-		contentCache: contentCache,
-		src:          *o,
+		bucket:         bucket,
+		mtimeClock:     mtimeClock,
+		id:             id,
+		name:           name,
+		attrs:          attrs,
+		localFileCache: localFileCache,
+		contentCache:   contentCache,
+		src:            *o,
 	}
 
 	f.lc.Init(id)
 
 	// Set up invariant checking.
 	f.mu = syncutil.NewInvariantMutex(f.checkInvariants)
 
-	if localFileCache {
-		// The gcs object is cached as local temp file
-		f.ensureContent(context.Background())
-	}
+	// if localFileCache {
```

Comment:
Remove the code that has been commented out.

Reply:
This is mistakenly commented out.  Thanks for catching.